src/test/kotlin/com/terraformation/backend/api/ControllerIntegrationTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.terraformation.backend.api
```

Comment:
This is taken directly from [here](https://github.com/terraware/pdd-backend/blob/main/server/src/test/kotlin/com/terraformation/pdd/ControllerIntegrationTest.kt)